yarn.lock

```diff
@@ -2008,6 +2008,7 @@
     eslint-scope "^5.0.0"
     eslint-utils "^2.0.0"
 
+
```

Comment:
The extra line should be removed, since `yarn.lock` shouldn't be modified directly by ourselves.